tools/gulp/packages.ts

```diff
@@ -0,0 +1,9 @@
+import {BuildPackage, buildConfig} from 'material2-build-tools';
+import {join} from 'path';
+
+export const cdkPackage = new BuildPackage('cdk');
+export const materialPackage = new BuildPackage('material', [cdkPackage]);
+export const examplesPackage = new BuildPackage('material-examples', [materialPackage, cdkPackage]);
+
+// To avoid refactoring of the project the material package will map to the source path `lib/`.
+materialPackage.packageRoot = join(buildConfig.packagesDir, 'lib');
```

Comment:
What do you you think of making the second constructor arg a config / options object so that it looks like
```ts
export const materialPackage = new BuildPackage('material', {
  deps: [cdkPackage],
  root: join(buildConfig.packagesDir, 'lib'),
});
```
?